src/login/login.c

```diff
@@ -1368,6 +1368,11 @@ static void login_auth_failed(struct login_session_data *sd, int result)
 	lclif->auth_failed(fd, ban_time, result);
 }
 
+// Generates auth token for web service
+static void login_generate_token(struct login_session_data* sd, unsigned char *auth_token) {
```

Comment:
at function start ``{`` should be on new line

Reply:
Fixed.